packages/api-explorer/package.json

```diff
@@ -68,15 +68,15 @@
   },
   "dependencies": {
     "@looker/extension-utils": "0.1.0",
```

Comment:
I think we need a prior PR to switch this to
```suggestion
    "@looker/extension-utils": "^0.1.0",
```

Reply:
Thanks

Reply:
fixing with https://github.com/looker-open-source/sdk-codegen/pull/921 as well as some other miscellany.